docs/topics/api/abuse.rst

```diff
@@ -0,0 +1,40 @@
+=============
+Abuse Reports
+=============
+
+The following API endpoint covers abuse reporting
+
+-------------------
+Submitting a report
+-------------------
+
+.. _`abusereport-create`:
+
+The following API endpoint allows an abuse report to be submitted for an Add-on
+or user on addons.mozilla.org.  Authentication is not required, but is recommended
```

Comment:
Pinging @jvillalobos because requiring authentication to report abuse would likely:

1. lessen spam
2. encourage user sign-in/account creation
3. improve quality of abuse reports/follow-up

the only downside being it's harder to report. But we make reviews (arguably more important) require sign-in. Why not do it here? I know spam is a HUGE issue for this right now and I feel this would help a lot AND improve quality.